app/src/main/java/org/wikipedia/talk/TalkPageSurvey.kt

```diff
@@ -0,0 +1,63 @@
+package org.wikipedia.talk
+
+import android.app.Activity
+import android.content.Context
+import android.widget.TextView
+import androidx.appcompat.app.AlertDialog
+import androidx.core.text.toSpannable
+import org.wikipedia.R
+import org.wikipedia.WikipediaApp
+import org.wikipedia.page.LinkMovementMethodExt
+import org.wikipedia.settings.Prefs
+import org.wikipedia.util.CustomTabsUtil
+import org.wikipedia.util.StringUtil
+
+object TalkPageSurvey {
+
+    fun shouldShowSurvey(): Boolean {
+        return Prefs.talkPageSurveyOverride
```

Comment:
This no longer takes into account all the logic for which user segments we're showing the survey.  As specified in the [task](https://phabricator.wikimedia.org/T291850):
```
Narrowed to target demos with required translations:

* English Wikipedia users/editors in India and Nigeria
* Arabic and French Wikipedia users/editors from Morocco, Egypt, DR Congo and Mali
* Hindi Wikipedia users/editors
* Indonesian Wikipedia users/editors
* Japanese Wikipedia
```

All of this logic was already in the `TalkPageSurveyHelper` class. Please make use of it here.